ckanext/switzerland/harvester/base_sbb_harvester.py

```diff
@@ -97,7 +97,6 @@ class BaseSBBHarvester(HarvesterBase):
         "accrual_periodicity": "",
         # --------------------------------------------------------------------------
         "description": {"fr": "", "en": "", "de": "", "it": ""},
-        "notes": {"fr": "", "en": "", "de": "", "it": ""},
```

Comment:
We never want a value here - it's a CKAN default field, not in our schema.

Reply:
thank you for explaining!